libobs/obs.h

```diff
@@ -1691,6 +1691,9 @@ EXPORT void obs_sceneitem_get_info(const obs_sceneitem_t *item,
 				   struct obs_transform_info *info);
 EXPORT void obs_sceneitem_set_info(obs_sceneitem_t *item,
 				   const struct obs_transform_info *info);
+EXPORT void obs_sceneitem_vecdiff_alignment(const obs_sceneitem_t *item,
+					    struct vec2 *v, uint32_t align_1,
```

Comment:
would use "offset" or "vecdiff" instead of v and add it to the documentation

Reply:
Overlooked the documentation of the obs file. Done